### PR TITLE
Document Sensu Core 1.2.0

### PR DESCRIPTION
**Pull request by portertech**

### CODE REVIEW

File: docs/1.2/enterprise/integrations/puppet.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+### Example(s)
+
+The following is an example global configuration for the `puppet` enterprise
+handler (integration).
+
+~~~ json
+{
+  "puppet": {
+    "endpoint": "https://10.0.1.22:8081/pdb/query/v4/nodes/",
```
Comment (cwjohnston) [inline docs/1.2/enterprise/integrations/puppet.md:39]: Looks like sed has been matching/changing IP addresses like this one between 1.x iterations of the docs. I don't think it matters wrt documentation content.

**Comment by cwjohnston** [comment]
I noticed an unintended change to the example IP address used in the puppetdb enterprise integration docs but I don't think of consequence.

**Comment by cwjohnston** [comment]
Silencing reference doc doesn't describe new `begin` parameter, although it's been added to the API reference.

**Comment by cwjohnston** [comment]
Results API reference doc does not illustrate history data as added by 1.2

File: docs/1.2/enterprise/integrations/puppet.md (hunk 1 above)
Comment (portertech) [reply docs/1.2/enterprise/integrations/puppet.md:39]: Yeah, I did see this. I did ignore it.

**Comment by portertech** [comment]
@cwjohnston thank you for the review! I think I've addressed the issues you raised. We can always tweak things going forward :+1:

**Pull request merged**